apps/theme/components/Group/Group.tsx

```diff
@@ -26,7 +26,7 @@ export const Group = ({
   colorScale,
   namespace,
 }: GroupProps) => {
-  const appearance = useThemeStore((state) => state.appearance);
+  const colorScheme = useThemeStore((state) => state.appearance);
```

Comment:
You should change this is the query params as well. 
See `useThemeParams.tsx`

Reply:
updated all reference to `apperance` without going into breaking-changes (query string).